src/managing-deis/platform-ssl.md

```diff
@@ -1,3 +1,58 @@
 # Platform SSL
 
-TODO (bacongobbler): rewrite for v2
+SSL/TLS is the standard security technology for establishing an encrypted link between a web server
+and a browser. This link ensures that all data passed between the web server and browsers remain
+private and integral.
+
+To enable SSL for your cluster and all apps running upon it, you can add an SSL key to your load
+balancer. You must either provide an SSL certificate that was registered with a CA or provide
+[your own self-signed SSL certificate](../reference-guide/creating_self_signed_ssl.md).
+
+## Installing SSL on a Load Balancer
+
+On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
+itself. Any communication inbound to the cluster will be encrypted while the internal components
+of Deis will still communicate over HTTP.
+
+To enable SSL, you will need to open port 443 on the load balancer and forward it to port 80 on the
+routers. For AWS, you'll also need to add port 443 in the security group settings for your load
+balancer.
+
```

Comment:
> So modifying the load balancer to terminate SSL is really a matter of:
> 
>     Swap the listener on 443 to use HTTPS instead of TCP.
>     Swap the back end of that listener to use HTTP and whatever back end node port the listener on 80 is already using.
>     Install the cert.

Perfect! That's all I was trying to convey. I'd like to use the host port, however I feel like we cannot rely on this behaviour. For example, what if a user's application (or another third party app) binds to port 80 on the host? Given that there's enough nodes to schedule that pod, a load balancer pointing at the minion's host port 80 would then cycle this user's app into the routing table because it is listening on port 80, passing the health check. I think it's best to play around with the load balancer that kubernetes has handed us :)

> I think my point is that there's actually a dizzying number of options for how to do this.

That is certainly true. There are many networking topologies; I guess it depends on how far down the network stack we want to stop (kube-proxy vs host port). I tend to lean towards the former, even if there's a performance hit for now.

> Part of me is tempted to drop this section entirely and have the docs deal only with terminating SSL at the router since that requires less setup and is more secure. If anyone really wants to terminate at the load balancer, they can probably figure it out for themselves.

We said that initially, then some users care whether SSL is terminated at the load balancer vs. termination at the routers. I think it's best to at least point users in the right direction. :)